examples/pxScene2d/src/shell.js

```diff
@@ -127,6 +127,30 @@ if( scene.capabilities != undefined && scene.capabilities.graphics != undefined
     });
 }
 ////
+
+////
+if( scene.capabilities != undefined && scene.capabilities.graphics != undefined && scene.capabilities.graphics.dirtyRect == 1) {
```

Comment:
Please remove the capabilities check here since it's not needed.  This internal code will always have dirty rectangle support.  This is a good check to have in external Spark apps.  Should this if statement check to see if dirty rects are enabled instead of a capability check?